contracts/MeanFinance/MeanFinanceTarget.sol

```diff
@@ -4,134 +4,61 @@ pragma solidity ^0.8.13;
 import {IConnext} from "@connext/interfaces/core/IConnext.sol";
 import {IXReceiver} from "@connext/interfaces/core/IXReceiver.sol";
 import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import "./MeanFinanceAdapter.sol";
-import "../Uniswap/UniswapAdapter.sol";
 
-contract MeanFinanceTarget is MeanFinanceAdapter, UniswapAdapter {
-    // The Connext contract on this domain
-    IConnext public immutable connext;
+import "./MeanFinanceAdapter.sol";
+import "../Uniswap/UniswapV3ForwarderXReceiver.sol";
```

Comment:
should use the `import {} from ` syntax for the three imports lacking it

Reply:
https://github.com/connext/connext-integration/pull/2/commits/e4718e351f7a63e46a02a3f8e992015548dbbb6a